lib/components/resinio/sdk.js

```diff
@@ -40,16 +40,27 @@ exports.getAllSupportedOSVersions = (deviceType) => {
 
 exports.downloadDeviceTypeOS = async (deviceType, version, destination) => {
   const stream = await resin.models.os.download(deviceType, version)
+  const bar = new visuals.Progress('Download')
+
+  let report = (data) => {
+    if (Math.floor(data.percentage) % 5 === 0) {
+      console.log(`Downloading OS for ${deviceType} (${version}): ${data.percentage.toFixed(2)}%`)
+    }
+  }
+
   if (!process.env.CI) {
-    const bar = new visuals.Progress('Download')
-    stream.on('progress', (data) => {
+    report = (data) => {
       bar.update({
         percentage: data.percentage,
         eta: data.eta
       })
-    })
+    }
   }
 
+  stream.on('progress', (data) => {
+    report(data)
```

Comment:
Could we put the different report logic depending on `process.env.CI` inside here?